pkg/config/redisstore.go

```diff
@@ -89,6 +106,7 @@ func (rs *redisStore) Set(key, value string) (index int, err error) {
 }
 
 func (rs *redisStore) List(key string, recurse bool) (keys []string, index int, err error) {
+	keys = make([]string, rs.listLength)
```

Comment:
should be `make([]string, 0, rs.listLength)` ?
Otherwise keys will have first 5 elements as empty right?